src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,85 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Calculator {
+
+    private final String expression;
+
+    public Calculator(String expression){
+        validInput(expression);
+        this.expression = expression;
+    }
+
+    public String getExpression() {
+        return expression;
+    }
+
+    public int calculate() {
+
+        String[] splitExpression = splitInput(expression);
+        validInteger(splitExpression[0]);
+
+        AtomicInteger result = new AtomicInteger(Integer.parseInt(splitExpression[0]));
+        List<Integer> valueList = getValue(splitExpression);
+        List<String> operatorList = getOperator(splitExpression);
```

Comment:
```suggestion
        String[] splitExpression = splitInput(expression);
        
        List<Integer> valueList = getValue(splitExpression);
        List<String> operatorList = getOperator(splitExpression);
       
        AtomicInteger result = new AtomicInteger(valueList.[0]);
```
위 처럼 선언하는것이 효율적일것 같아요 !
getValue() 메소드에서 유효한 숫자인지 판단하는 부분도 들어갈수 있을것 같습니다 !